.gitignore

```diff
@@ -56,4 +56,5 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+.env.example

```

Comment:
@suolevram Oh my bad, I thought this review was looking at the dockerignore file. Yeah, definitely don't commit the `.env` file.